src/components/activity/Navbar.js

```diff
@@ -0,0 +1,100 @@
+import styled from 'styled-components';
+import {Link, useMatch, useResolvedPath} from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <StyledNavigation>
+      <>
+        <main>
+          <header>
+            <img width={180} height={180} src="./images/logo.png" alt="logo" />
+            <h2>Structure your Day</h2>
+          </header>
+
+          <nav>
+            <CustomLink to="/">Homepage</CustomLink>
+            <CustomLink to="/listEntries">Einträge</CustomLink>
+            <CustomLink to="/todos">Todos</CustomLink>
+          </nav>
+        </main>
+      </>
+    </StyledNavigation>
+  );
+}
+
+function CustomLink({to, children, ...props}) {
+  const resolvePath = useResolvedPath(to);
+  const isActive = useMatch({path: resolvePath.pathname, end: true});
+
+  return (
+    <li className={isActive ? 'active' : ''}>
+      <Link to={to} {...props}>
+        {children}
+      </Link>
+    </li>
```

Comment:
Where is your list to wrap these `li` elements? As far as I can see, you use `CustomLink above inside a `nav`, but that's not allowed.